api/types/types.go

```diff
@@ -318,15 +318,16 @@ type ContainerJSONBase struct {
 	SizeRootFs      *int64 `json:",omitempty"`
 }
 
-// ContainerJSON is newly used struct along with MountPoint
+// ContainerJSON is a representation of a container state meant to be externally consumed (eg. through the API, by
+// swamrkit, etc...).
```

Comment:
Bit on the fence on the "to be externally consumed ........".


It being in the `api/` / `api/types` package should be a clue that it's part of the API (at least _most_ types in there should be part of the actual API (we still have some cleaning up to do).

Perhaps it's possible to describe what it's for (this is the type used by container _inspect_ correct?),

Ideally these types would also be in the `api/types/container` package (in which case it could become `api/types/container.InspectResponse` (or something along those lines).